templates/common/copilot-gpt-basic/appPackage/manifest.json.tpl

```diff
@@ -28,6 +28,9 @@
         "identity",
         "messageTeamMembers"
     ],
-    "copilotGpts" [],
+    "copilotGpts" [
+        "id": "gpt_1",
```

Comment:
maybe not gpt_1 as we are not going to call it GPT anymore?

Reply:
Yes, not just gpt_1, but also `copilotGps`. Once the naming is finalized, I will update everything together.